lib/index_column.c

```diff
@@ -744,6 +759,43 @@ static void
 grn_index_column_diff_process_token_id(grn_ctx *ctx,
                                        grn_index_column_diff_data *data)
 {
+  {
+    void *value = NULL;
+    int added = 0;
+    const grn_id token_counter_id = grn_hash_add(ctx,
+                                                 data->current.token_counters,
+                                                 &(data->current.token_id),
+                                                 sizeof(grn_id),
+                                                 &value,
+                                                 &added);
+    if (token_counter_id == GRN_ID_NIL) {
+      grn_rc rc = ctx->rc;
+      if (rc == GRN_SUCCESS) {
+        rc = GRN_UNKNOWN_ERROR;
+      }
+      char message[GRN_CTX_MSGSIZE];
+      grn_strcpy(message, GRN_CTX_MSGSIZE, ctx->errbuf);
+      ERR(rc,
+          "[index-column][diff] failed to add a token counter: "
+          "<%.*s>: %u: %s",
+          data->index.name_size,
+          data->index.name,
+          data->current.token_id,
+          message);
+      return;
+    }
+
+    uint32_t *token_counter = value;
+    if (added) {
+      *token_counter = 0;
+    } else {
+      (*token_counter)++;
+    }
+
+    if (*token_counter >= GRN_II_MAX_TF) {
+      return;
+    }
+  }
```

Comment:
空行を入れて論理的に違う処理感を出しましょうか。

```suggestion
  }

```

Reply:
fix: [b052e24](https://github.com/groonga/groonga/pull/2035/commits/b052e24ae4ad9da6e88654f1ef90fa5b5b543324)